CHANGELOG.md

```diff
@@ -4,13 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Multi-version plotting capabilities for generated example plots
+- JSON file for plotting themes
+- Smoke testing in relevant tox environments
+- Smoke testing for `botorch_analytical` example
+
 ## [0.7.3] - 2024-02-09
 ### Added
 - Copy button for code blocks in documentation
 - `mypy` for campaign, constraints and telemetry
 - Top-level example summaries
 - `RecommenderProtocol` as common interface for `Strategy` and `Recommender`
-- `SubspaceDiscrete.from_simplex` convenience constructor
```

Comment:
I think this one got messed up during rebase!?

Reply:
Fixed.